docs/api/geolocate.rst

```diff
@@ -156,11 +156,11 @@ signalStrength
 
 
 Cell Tower Fields
-~~~~~~~~~~~~~~~~~
+-----------------
 
 radioType
-    The type of radio network. One of `gsm`, `wcdma` or `lte`.
-    This is a custom extension to the GLS API, which only defines the
+    The type of radio network. One of ``gsm``, ``wcdma``, or ``lte``.
```

Comment:
Yeah... I almost didn't do this because it felt like I was imposing my style on things. Now that I'm looking at it, those should all be glossary term links.

Reply:
Future someone should do a pass on linking things to the glossary.